core/spec/models/spree/order_inventory_spec.rb

```diff
@@ -164,9 +164,9 @@
 
       it 'should destroy backordered units first' do
         allow(shipment).to receive_messages(inventory_units_for_item: [
-          mock_model(Spree::InventoryUnit, :variant_id => variant.id, :state => 'backordered'),
-          mock_model(Spree::InventoryUnit, :variant_id => variant.id, :state => 'on_hand'),
-          mock_model(Spree::InventoryUnit, :variant_id => variant.id, :state => 'backordered')
+          mock_model(Spree::InventoryUnit, variant_id: variant.id, state: 'backordered', shipped?: false),
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.